assets/training/finetune_acft_hf_nlp/components/pipeline_components/text_classification/spec.yaml

```diff
@@ -640,7 +640,7 @@ jobs:
       pytorch_model_folder: '${{parent.outputs.pytorch_model_folder}}'
   ft_nlp_model_converter:
     type: command
-    component: azureml:ft_nlp_model_converter:0.0.49
+    component: azureml:ft_nlp_model_converter:0.0.50
```

Comment:
ft_nlp_model_converter has older environment version, but finetuning has latest transformers. This will cause issues right?

Reply:
Added new model convertor for text classification

Reply:
Shouldn't this be text_classification_model_converter

Reply:
fixed